src/components/KMethodBadge/KMethodBadge.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <KBadge
+    :background-color="backgroundColor || methodBadgeColors[method].backgroundColor"
+    :border-color="borderColor || methodBadgeColors[method].borderColor"
+    class="k-method-badge"
+    :class="[ `method-${method}`, isToggle ? 'k-method-toggle' : '']"
+    :color="color || methodBadgeColors[method].color"
+    :is-bordered="isToggle"
+    max-width="auto"
+    :shape="shape === 'rectangular' ? 'rectangular' : 'rounded'"
+    v-on="isToggle ? { click: toggleValue } : {}"
```

Comment:
question: why `v-on` vs `@click`?

Reply:
Because if we have `@click` (even with undefined event handler function) KBadge automatically makes the element clickable (under the hood KBadge checks if `onClick` attribute is present on the element). So as a way to go around that, I have a `v-on` that only adds `click` event handler if it needs to.

Reply:
Wouldn't this unbind _any_ event listener if `isToggle` is false? I'm not sure that's ideal.

Reply:
Done